ga4gh/frontend.py

```diff
@@ -194,6 +194,7 @@ def configure(configFile=None, baseConfig="ProductionConfig",
     TODO Document this critical function! What does it do? What does
     it assume?
     """
+    requests.packages.urllib3.disable_warnings()
```

Comment:
But we're not actually in testing mode when we do this though (I think), so it won't do any good. I think the simplest option is to add a server CLI option `--disable-urllib-warnings` and just call `` requests.packages.urllib3.disable_warnings()`` directly from `cli.py` if it's present. It's a hack, but it's generally handy to be able to turn off this nagging when you're developing. We might add it to the client too, come to think of it.